edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Amqp/CbsNode.cs

```diff
@@ -148,8 +148,8 @@ internal static (string token, string audience) ValidateAndParseMessage(string i
 
             try
             {
-                SharedAccessSignature sharedAccessSignature = SharedAccessSignature.Parse(iotHubHostName, token);
-                return (token, sharedAccessSignature.Audience);
+                string audience = message.ApplicationProperties.Map[CbsConstants.PutToken.Audience] as string;
```

Comment:
Tests?

Reply:
I updated the existing test collateral to use the audience ApplicationProperty.